spec/calendar.html

```diff
@@ -804,191 +794,200 @@ <h1>Temporal.Calendar.prototype.dateUntil ( _one_, _two_, _options_ )</h1>
     </emu-clause>
 
     <emu-clause id="sec-temporal.calendar.prototype.year">
-      <h1>Temporal.Calendar.prototype.year ( _dateOrDateTime_ )</h1>
+      <h1>Temporal.Calendar.prototype.year ( _temporalDateLike_ )</h1>
       <p>
         An ECMAScript implementation that includes the ECMA-402 Internationalization API must implement the `year` method as specified in the ECMA-402 specification.
         If an ECMAScript implementation does not include the ECMA-402 API the following specification of the `year` method is used.
       </p>
       <p>
-        The `year` method takes one argument _dateOrDateTime_.
+        The `year` method takes one argument _temporalDateLike_.
         The following steps are taken:
       </p>
       <emu-alg>
         1. Let _calendar_ be the *this* value.
         1. Perform ? RequireInternalSlot(_calendar_, [[InitializedTemporalCalendar]]).
         1. Assert: _calendar_.[[Identifier]] is *"iso8601"*.
-        1. Return ? ISOYear(_dateOrDateTime_).
+        1. If Type(_temporalDateLike_) is not Object or _temporalDateLike_ does not have an [[InitializedTemporalDate]] or [[InitializedTemporalYearMonth]] internal slot, then
+          1. Set _temporalDateLike_ to ? ToTemporalDate(_temporalDateLike_).
```

Comment:
nit:
```suggestion
          1. Return ! ISOYear(? ToTemporalDate(_temporalDateLike_)).
```
to avoid reusing the variable name?